### PR TITLE
Add new playbook "Send-UnhealthyAzureArcResourceAlert"

### PR DESCRIPTION
**Pull request by mlaraibkhan**
Required items, please complete
   
   Change(s):
   - Add new playbook "Send-UnhealthyAzureArcResourceAlert"

   Reason for Change(s):
   - Contributing new playbook

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by mlaraibkhan** [comment]
@microsoft-github-policy-service agree

File: Playbooks/Send-UnhealthyAzureArcResourceAlert/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,215 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
```
Comment (v-prasadboke) [inline Playbooks/Send-UnhealthyAzureArcResourceAlert/azuredeploy.json:4]: Hello @mlaraibkhan please fill the required metadata

Comment (v-prasadboke) [reply Playbooks/Send-UnhealthyAzureArcResourceAlert/azuredeploy.json:4]: You can refer to this playbook for metadata Solutions/AWSAthena/Playbooks/AWSAthenaPlaybooks/AWSAthena-GetQueryResults/azuredeploy.json

Comment (mlaraibkhan) [reply Playbooks/Send-UnhealthyAzureArcResourceAlert/azuredeploy.json:4]: Sure Thanks for pointing that out. I added the required metadata.

File: Playbooks/Send-UnhealthyAzureArcResourceAlert/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+##Edit *Send an Email* Action
+Replace <email-id@domain.com> to your own choice of email address.
+![screenshot](./images/email.png)
+
+
+# Deploy to Azure
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FSend-UnhealthyAzureArcResourceAlert%2Fazuredeploy.json" target="_blank">
+<img src="https://aka.ms/deploytoazurebutton""/>
```
Comment (v-prasadboke) [inline Playbooks/Send-UnhealthyAzureArcResourceAlert/README.md:23]: Hello @mlaraibkhan please remove extra " from the img src  **button""/>**

Comment (mlaraibkhan) [reply Playbooks/Send-UnhealthyAzureArcResourceAlert/README.md:23]: Removed

Comment (v-prasadboke) [reply Playbooks/Send-UnhealthyAzureArcResourceAlert/README.md:23]: Hello @mlaraibkhan this was not i meant, i requested to remove extra " from line 23 which is . If you look at deloytoazurebutton""  there is an extra" at the end. 
I asked to remove that.
And please deploy to azuregov button again

Comment (mlaraibkhan) [reply Playbooks/Send-UnhealthyAzureArcResourceAlert/README.md:23]: Ops my bad, corrected

File: Playbooks/Send-UnhealthyAzureArcResourceAlert/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,213 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "Send Unhealthy Azure Arc Resource Alert",
+        "description":  " Sentinel logic app designed to send an email alert when an unhealthy Azure Arc resource is detected. The logic app is implemented using Azure Logic Apps and utilizes Azure Monitor Logs and Office 365 connectors for its functionality.",
+        "prerequisites":  "Office 365 Account",
```
[the remark this answers is not in the document]
Comment (mlaraibkhan) [reply Playbooks/Send-UnhealthyAzureArcResourceAlert/azuredeploy.json]: Done!

File: Playbooks/Send-UnhealthyAzureArcResourceAlert/README.md
[the remark this answers is not in the document]
Comment (mlaraibkhan) [reply Playbooks/Send-UnhealthyAzureArcResourceAlert/README.md:1]: I've added prerequisite and post-deployment steps in the readme.

**Review by mlaraibkhan — COMMENTED** [review]
I have made the suggested changes to the files.

**Comment by v-prasadboke** [comment]
Thank you @mlaraibkhan for committing the changes. Will be investigating this commits.
You can expect an update till 25 Jul 2023

**Review by v-prasadboke — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**